src/ILCompiler.Compiler/src/Compiler/LibraryInitializers.cs

```diff
@@ -66,9 +66,6 @@ private void InitLibraryInitializers()
 
             foreach (var entry in s_assembliesWithLibraryInitializers)
             {
-                if (_isCppCodeGen && !entry.UseWithCppCodeGen)
```

Comment:
Could please clear the flag isCppCodeGen flag [here]( https://github.com/dotnet/corert/blob/master/src/ILCompiler.Compiler/src/Compiler/LibraryInitializers.cs#L28), and delete link to the fixed issues [here]( https://github.com/dotnet/corert/blob/master/src/ILCompiler.Compiler/src/Compiler/LibraryInitializers.cs#L45) instead?

I think the flag to filter based on isCppCodeGen can stay around; even though it is unused right now. We will likely want different implementations of some parts for it eventually; or it may useful to workaround CppCodeGen bugs.